plotly/Cargo.toml

```
@@ -41,7 +41,7 @@ wasm-bindgen-futures = { version = "0.4", optional = true }
 [dev-dependencies]
 csv = "1.1.6"
 image = "0.24.4"
-itertools = "0.10.3"
+itertools = "0.12.1"
```

```suggestion
itertools = ">=0.10, <0.13"
```
I suggest permitting versions 0.10, 0.11, or 0.12.